nomad/structs/acl.go

```diff
@@ -474,6 +474,16 @@ func (a *ACLToken) UnmarshalJSON(data []byte) (err error) {
 	return nil
 }
 
+func (a *ACLToken) Sanitize() *ACLToken {
+	if a == nil {
+		return nil
+	}
+
+	out := a.Copy()
+	out.SecretID = ""
```

Comment:
Why not `"<redacted>"`? I think Node only does `""` because there's no reason for humans to even need to know that field exists. It's an internal implementation detail.

I'm fine with ACLToken going either way, and I suppose there's something to be said for consistency.

Reply:
I didn't want to change the format of the objects we're emitting in the event stream (at least not without thinking about that).